firmware/Core/Src/unit_tests/test_ant_commands.c

```diff
@@ -0,0 +1,21 @@
+#include "unit_tests/unit_test_helpers.h"
+#include "unit_tests/test_assorted_prototypes.h"
+#include<antenna_deploy_drivers/ant_commands.h>
+
+uint8_t TEST_EXEC__ANT_CMD_measure_temp() {
+    int16_t result =  ANT_CMD_convert_temp_measurement_to_centi_degree_celsius(647);
+    
+    TEST_ASSERT(100 < result && result  < 200 );
```

Comment:
Would recommend using more parenthesis around `(100 < result) && (...)`. 

It's normally fine, until it's not. C has a few quirks in the bitwise and boolean-like operations where the order of operations isn't quite what you'd expect. Probablyyy no need to change here though